gamedata.cpp

```diff
@@ -310,7 +310,7 @@ ItemType id[] =
 	 0,
 	 "FISH",1,1,1, {{-1,0}, {-1,0},{-1,0},{-1,0}},
 	 NULL,0,0, {{-1,0},{-1,0},{-1,0},{-1,0}},
-	 1, IT_NORMAL | IT_FOOD, 15,0,
+	 1, IT_NORMAL | IT_FOOD, 20,0,
```

Comment:
IMO it is better to change such values in the ruleset folder via `Modify*` methods. Then there will be no need to change it back if we will merge changes back to the upstream Atlantis repo.